docs/getting_started/using_modin/using_modin_cluster/using_modin_ray_cluster.rst

```diff
@@ -0,0 +1,132 @@
+local development and cluster execution. Users are not required to think about
+how many workers exist or how to distribute and partition their data;
+Modin handles all of this seamlessly and transparently.
+
+.. note::
+   You can also use a Jupyter notebook, but you need to deploy a Jupyter server 
+   on the remote cluster head node and connect to it.
+
+.. image:: ../../../img/modin_cluster.png
+   :alt: Modin cluster
+   :align: center
+
+Extra requirements for AWS authentication
+-----------------------------------------
+
+First of all, install the necessary dependencies in your environment:
+
+.. code-block:: bash
+
+   pip install boto3
+
+The next step is to setup your AWS credentials. One can set  ``AWS_ACCESS_KEY_ID``, 
+``AWS_SECRET_ACCESS_KEY`` and ``AWS_SESSION_TOKEN`` environment variables or  
+just run the following command:
+
+.. code-block:: bash
+
+   aws configure
+
+Starting and connecting to the cluster
+--------------------------------------
+
+This example starts 1 head node (m5.24xlarge) and 5 worker nodes (m5.24xlarge), 576 total CPUs.
+You can check the `Amazon EC2 pricing`_ .
+
+You can manually create AWS EC2 instances and configure them or just use the `Ray CLI`_ to 
+create and initialize a Ray cluster on AWS using `Modin's Ray cluster setup config`_ .
+You can read more about how to modify the file on `Ray's autoscaler options`_ .
+
+More details on how to launch a Ray cluster can be found on `Ray's cluster docs`_.
+
+To start up the Ray cluster, run the following command in your terminal:
+
+.. code-block:: bash
+
+   ray up modin-cluster.yaml
+
+Once the head node has completed initialization, you can optionally connect to it by running the following command.
+
+.. code-block:: bash
+
+   ray attach modin-cluster.yaml
+
+To exit the ssh session and return back into your local shell session, type:
+
+.. code-block:: bash
+
+   exit
+
+Executing in a cluster environment
+----------------------------------
+
+.. note::
+   Be careful when using the `Ray client`_ to connect to a remote cluster.
+   We don't recommend this connection mode, beacuse it may not work. Known bugs:
+   - https://github.com/ray-project/ray/issues/38713,
+   - https://github.com/modin-project/modin/issues/6641.
+
+Modin lets you instantly speed up your workflows with a large data by scaling pandas
+on a cluster. In this tutorial, we will use a 12.5 GB `big_yellow.csv` file that was
+created by concatenating a 200MB `NYC Taxi dataset`_ file 64 times. Preparing this
+file was provided as part of our `Modin's Ray cluster setup config`_.
+
+If you want to use the other dataset, you should provide it to each of
+the cluster nodes with the same path. We recomnend doing this by customizing the
+`setup_commands` section of the [configuration file](https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/pandas_on_ray/cluster/modin-cluster.yaml).
```

Comment:
```suggestion
``setup_commands`` section of the [configuration file](https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/pandas_on_ray/cluster/modin-cluster.yaml).
```
The link to `configuration file` is not displayed correctly in the docs. Please recheck.